[2024-06-24 14:38:56,040] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_dag_with_python_operator_v07.get_name 2023-10-15T00:00:00+00:00 [queued]>
[2024-06-24 14:38:56,844] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_dag_with_python_operator_v07.get_name 2023-10-15T00:00:00+00:00 [queued]>
[2024-06-24 14:38:56,869] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 14:38:56,881] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-06-24 14:38:56,887] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-06-24 14:38:57,309] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): get_name> on 2023-10-15T00:00:00+00:00
[2024-06-24 14:38:57,620] {standard_task_runner.py:52} INFO - Started process 22009 to run task
[2024-06-24 14:38:57,994] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_dag_with_python_operator_v07', 'get_name', '2023-10-15T00:00:00+00:00', '--job-id', '8656', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmp3rby76ky', '--error-file', '/tmp/tmpiyt71gf2']
[2024-06-24 14:38:58,173] {standard_task_runner.py:77} INFO - Job 8656: Subtask get_name
[2024-06-24 14:38:59,768] {logging_mixin.py:104} INFO - Running <TaskInstance: our_dag_with_python_operator_v07.get_name 2023-10-15T00:00:00+00:00 [running]> on host 799b1e36e3f1
[2024-06-24 14:39:01,444] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_dag_with_python_operator_v07
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2023-10-15T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-15T00:00:00+00:00
[2024-06-24 14:39:02,130] {python.py:118} INFO - Done. Returned value was: None
[2024-06-24 14:39:02,524] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_v07, task_id=get_name, execution_date=20231015T000000, start_date=20240624T143856, end_date=20240624T143902
[2024-06-24 14:39:04,066] {local_task_job.py:188} WARNING - State of this instance has been externally set to success. Terminating instance.
[2024-06-24 14:39:04,511] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 22009
[2024-06-24 14:39:04,563] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-06-24 14:39:04,845] {process_utils.py:66} INFO - Process psutil.Process(pid=22009, status='terminated', exitcode=1, started='14:38:56') (22009) terminated with exit code 1
